hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/ClassAndFieldDefinitionTest.java

```diff
@@ -195,4 +196,16 @@ public void testFieldDef_equal_hashCode() throws Exception {
     public void testFieldDef_toString() throws Exception {
         assertNotNull(new FieldDefinitionImpl(0, "name", FieldType.BOOLEAN, portableVersion).toString());
     }
+
+    @Test(expected = HazelcastSerializationException.class)
+    public void testClassDefinitionBuilder_addingSameFieldTwice() {
+        new ClassDefinitionBuilder(1, 2, 1)
+                .addUTFField("name").addUTFField("name").build();
```

Comment:
invocation of `build()` can be removed since the exception is expected from the second `add*` method